development-docs/RELEASE.md

```diff
@@ -13,44 +13,45 @@ The release process should normally look like this:
 1. Create a release branch starting from the `main` one. The new release branch has to be named like `release-<Major>.<minor>.x`, for example `release-0.45.x` to be used for all patch releases for the 0.45.
 2. On the `main` git branch of the repository:
    * Update the versions to the next SNAPSHOT version using the `next_version` `make` target. For example to update the next version to `0.46.0-SNAPSHOT` run: `NEXT_VERSION=0.46.0-SNAPSHOT make next_version`.
-   * Update the product version in the `documentation/shared/attributes.adoc` file to the next version by setting the `ProductVersion` variable and also the previous version by setting the `ProductVersionPrevious` variable.
-   * Add a header for the new release to the `CHANGELOG.md` file
+   * Update the product version in the `documentation/shared/attributes.adoc` file to the next version by setting the `ProductVersion` variable and also the previous version the to one you are releasing by setting the `ProductVersionPrevious` variable.
+   * Add a header for the next release to the `CHANGELOG.md` file
 3. Move to the release branch and run `make clean`
-4. Use the `RELEASE_VERSION` environment variable to set the desired version
-   * Use always the GA version here (e.g. `0.45.0`) and not the RC version (e.g `0.45.0-rc1`)
-5. Run `RELEASE_VERSION=<desired version> make release`, for example `RELEASE_VERSION=0.45.0 make release`
+4. Run `RELEASE_VERSION=<desired version> make release`, for example `RELEASE_VERSION=0.45.0 make release`
```

Comment:
I ran `export RELEASE_VERSION` then realised the next step included the env var, so I think this would be clearer as one step